* Omit "license" keyword for now to be able to build with older py-setuptools.

* Provide the console scripts without their ".py" extension.

See also: https://github.com/pdfminer/pdfminer.six/issues/724

--- pyproject.toml.orig	2025-12-30 15:48:54 UTC
+++ pyproject.toml
@@ -7,7 +7,6 @@ readme = "README.md"
 dynamic = ["version"]
 description = "PDF parser and analyzer"
 readme = "README.md"
-license = "MIT"
 requires-python = ">=3.10"
 authors = [
     { name = "Yusuke Shinyama" },
@@ -68,8 +67,8 @@ script-files = [
 [tool.setuptools]
 packages = ["pdfminer"]
 script-files = [
-    "tools/pdf2txt.py",
-    "tools/dumppdf.py",
+    "tools/pdf2txt",
+    "tools/dumppdf",
 ]
 
 [tool.setuptools_scm]
